apps/teams-test-app/e2e-test-data/pages.json

```diff
@@ -25,6 +25,14 @@
       "expectedAlertValue": "returnFocus called with true",
       "expectedTestAppValue": "Current navigateForward state is true"
     },
+    {
+      "title": "returnFocus With Type API Call - Success",
+      "type": "callResponse",
+      "boxSelector": "#box_returnFocusWithType",
+      "inputValue": 2,
+      "expectedAlertValue": "returnFocus called with false",
```

Comment:
Is the difference here because you are testing that the "false" fallback is passed up to the host when they don't know about the new enum?

Reply:
well I initially thought to align what hub tests were - but considering there's a new arg in town - I changed the test to reflect that.